### PR TITLE
MOTD messages must be prefixed with colon

### PR DESCRIPTION
**Pull request by Gargaj**
As per https://tools.ietf.org/html/rfc2812#page-48

### CODE REVIEW

File: lib/gitter-adapter.js [hunk 1]
```diff
@@ -329,9 +329,9 @@ Adapter.prototype.messageOfTheDay = function() {
   var c = this.client;
 
   c.send(c.host, irc.reply.motdStart, c.nick, ':- Message of the Day -');
-  c.send(c.host, irc.reply.motd,      c.nick, 'Welcome To Gitter IRC!');
-  c.send(c.host, irc.reply.motd,      c.nick, 'Info at https://irc.gitter.im');
-  c.send(c.host, irc.reply.motd,      c.nick, 'Code at https://github.com/gitterHQ/irc-bridge');
+  c.send(c.host, irc.reply.motd,      c.nick, ':Welcome To Gitter IRC!');
```
Comment (MadLittleMods) [inline lib/gitter-adapter.js]: According to that spec, wouldn't it be prefixed with `:- ` ?

I am looking at 372 `RPL_MOTD`

Comment (Gargaj) [inline lib/gitter-adapter.js]: Technically, I suppose, yes. I can amend the PR.

**Comment by MadLittleMods** [comment]
:+1: Tested with HexChat.

Current:

```
* - Message of the Day -
* Welcome To Gitter IRC!
* Info at https://irc.gitter.im
* Code at https://github.com/gitterHQ/irc-bridge
* End of /MOTD command.
```

With this patch:

```
* - Message of the Day -
* - Welcome To Gitter IRC!
* - Info at https://irc.gitter.im
* - Code at https://github.com/gitterHQ/irc-bridge
* End of /MOTD command.
```

**Comment by MadLittleMods** [comment]
I merged this PR into `develop`

**Pull request closed without merging**